files/vehicles_ft.md

```diff
@@ -24,16 +24,39 @@ File MAY contain the following attributes:
 | `vehicle_length`		| Float of the vehicle length in feet.
 | `platform_height`		| Float of the platform height in inches.
 | `propulsion_type`		| String of the name of the propulsion type.  Possible values include:
-|	| diesel
-| | bio-diesel
-| | diesel-hybrid
-| | CNG
-|	| electric-trolly
+|	| `diesel`
+| | `bio-diesel`
+| | `diesel-hybrid`
+| | `CNG`
+|	| `electric-trolly`
 | `wheelchair_capacity`	| Integer of total capacity for wheelchairs on vehicle. Overrides value in trip file.  
-|	| Blank - indicates that it is unknown and is treated as infinite.  
-| | 0  - indicates that wheelchairs cannot access this vehicle.
-| | 1+ - number of wheelchairs that can be accommodated
+|	| `Blank` - indicates that it is unknown and is treated as infinite.  
+| | `0`  - indicates that wheelchairs cannot access this vehicle.
+| | `1+` - number of wheelchairs that can be accommodated
 | `bicycle_capacity`		| Integer representation of [non-folding] bicycles that can be accommodated.  
-|	| Blank - indicates that it is unknown and is treated as infinite unless the trip file says that it is not bicycle accessible.
-|	| 0  - indicates that bicycles cannot ride on this vehicle.
-|	| 1+ - number of bicycles that can be accommodated
+|	| `Blank` - indicates that it is unknown and is treated as infinite unless the trip file says that it is not bicycle accessible.
+|	| `0`  - indicates that bicycles cannot ride on this vehicle.
+|	| `1+` - number of bicycles that can be accommodated
+| `boarding_door` | String identifying the door(s) used for boarding
+| | `Blank` - indicates that it is unknown, assumed to be front door boarding
+| | `front` - indicates boarding is only allowed at the front door
+| | `all` - indicates boarding is allowed at all doors
+| `fare_payment_method` | method of payment accepted in addition to cash fare at farebox
+| | `none`
+| | `visual_inspection`
+| | `single_ticket_token`
+| | `exact_change`
+| | `ticket_validator`
+| | `magstripe_card`
+| | `smart_card`
+| | `user_defined`
+| `boarding_height` | 
+| | `level`
+| | `stairs`
+| | `steep_stairs` |
+| `percent_using_farebox` | Floating point number between 0 and 1 denoting the percent of people boarding who pay a cash fare at the farebox
```

Comment:
@e-lo good point.  It might also vary by stop, maybe we could include it in stop_times_ft and routes_ft, with the former overriding the latter if it's provided?